### PR TITLE
Polish translation of Tour of Scala: case classes

### PR DESCRIPTION
**Pull request by kamilduda**
Make Polish version be in line with the English one

### CODE REVIEW

**Comment by kamilduda** [comment]
@jstefanska Could you review this in your spare time?

**Comment by SethTisue** [comment]
asked on Gitter (https://gitter.im/scala/scala?at=5c6f06255df86a456871a7de) for a reviewer, if that doesn't work we can try Discourse next

File: _pl/tour/case-classes.md [hunk 1]
```diff
@@ -6,128 +6,70 @@ discourse: false
...
-num: 10
+num: 11
 language: pl
 next-page: pattern-matching
 previous-page: multiple-parameter-lists
 ---
 
-Scala wspiera mechanizm _klas przypadków_. Klasy przypadków są zwykłymi klasami z dodatkowymi założeniami:
+Scala wspiera mechanizm _klas przypadków_ (ang. case class).
+Są one zwykłymi klasami z dodatkowymi założeniami przez które przejdziemy.
```
Comment (zygfryd) [inline _pl/tour/case-classes.md]:
```suggestion
Są one zwykłymi klasami z dodatkowymi założeniami, przez które przejdziemy.
```

File: _pl/tour/case-classes.md [hunk 2]
```diff
@@ -6,128 +6,70 @@ discourse: false
...
+Klasy przypadków idealnie nadają sie do modelowania niezmiennych (niemutowalnych) danych.
```
Comment (zygfryd) [inline _pl/tour/case-classes.md]:
```suggestion
Klasy przypadków idealnie nadają się do modelowania niezmiennych (niemutowalnych) danych.
```

File: _pl/tour/case-classes.md [hunk 3]
```diff
@@ -6,128 +6,70 @@ discourse: false
...
-Tworzenie obiektu jest bardzo proste: (Zwróć uwagę na to, że słowo `new` nie jest wymagane)
+case class Book(isbn: String)
 
-```tut
-val emailFromJohn = Email("john.doe@mail.com", "Greetings From John!", "Hello World!")
+val frankenstein = Book("978-0486282114")
 ```
 
-Parametry konstruktora klasy przypadków są traktowane jako publiczne wartości i można się do nich odwoływać bezpośrednio:
+Zauważ, że słowo kloszowe `new` nie było konieczne do stworzenia instancji klasy przypadku `Book`.
```
Comment (zygfryd) [inline _pl/tour/case-classes.md]:
```suggestion
Zauważ, że słowo kluczowe `new` nie było konieczne do stworzenia instancji klasy przypadku `Book`.
```

File: _pl/tour/case-classes.md [hunk 4]
```diff
@@ -6,128 +6,70 @@ discourse: false
...
+val message1 = Message("guillaume@quebec.ca", "jorge@catalonia.es", "Ça va ?")
 
-Zamiast tego możesz utworzyć kopię używając metody `copy`:
-
-```tut
-val editedEmail = emailFromJohn.copy(title = "I am learning Scala!", body = "It's so cool!")
-
-println(emailFromJohn) // wypisuje "Email(john.doe@mail.com,Greetings From John!,Hello World!)"
-println(editedEmail) // wypisuje "Email(john.doe@mail.com,I am learning Scala,It's so cool!)"
+println(msg.senasdfasdder)  // wypisze guillaume@quebec.ca
```
Comment (zygfryd) [inline _pl/tour/case-classes.md]:
```suggestion
println(message1.sender)  // wypisze guillaume@quebec.ca
```

File: _pl/tour/case-classes.md [hunk 5]
```diff
@@ -6,128 +6,70 @@ discourse: false
...
+message1.sender = "travis@washington.us"  // ten wiersz nie skompiluje się
 ```
 
-Dla każdej klasy przypadku kompilator Scali wygeneruje metodę `equals` implementującą strukturalne porównanie obiektów oraz metodę `toString`. Przykład:
+Nie można ponownie przydzielić wartości do `message1.sender`, ponieważ jest to `val` (stała).
+Jako alternatywa, w klasach przypadków można używać `var`, jednak odradzamy temu.
```
Comment (zygfryd) [inline _pl/tour/case-classes.md]:
```suggestion
Alternatywnie, w klasach przypadków można też używać `var`, jednak stanowczo tego odradzamy.
```

File: _pl/tour/case-classes.md [hunk 6]
```diff
@@ -6,128 +6,70 @@ discourse: false
...
-val someSms = SMS("12345", "Are you there?")
-val someVoiceRecording = VoiceRecording("Tom", "voicerecording.org/id/123")
-
-println(showNotification(someSms)) // Wypisuje "You got an SMS from 12345! Message: Are you there?"
-println(showNotification(someVoiceRecording)) // Wypisuje "you received a Voice Recording from Tom! Click the link to hear it: voicerecording.org/id/123"
-```
+## Kopiowanie
 
-Poniżej bardziej skomplikowany przykład używający `if` w celu określenia dodatkowych warunków dopasowania:
+Możliwe jest stworzenie płytkiej kopii (ang. shallow copy) instancji klasy przypadków używając metody `copy`.
```
Comment (zygfryd) [inline _pl/tour/case-classes.md]:
```suggestion
Możliwe jest stworzenie płytkiej kopii (ang. shallow copy) instancji klasy przypadku używając metody `copy`.
```

**Comment by kamilduda** [comment]
All fixed, thanks for the review! 😃

**Comment by SethTisue** [comment]
sorry for the slow merge — thank you!

**Pull request merged**